src/cryptography/x509.py

```diff
@@ -237,6 +237,57 @@ def __repr__(self):
                 "path_length={0.path_length})>").format(self)
 
 
+class KeyUsage(object):
+    def __init__(self, usages):
```

Comment:
Is passing in a list of bools really the best option here? I wonder if defaulting self._usages to a list filled with False would be the cleaner option here.